Plots/Polygons/NCL_polyg_18.py

```diff
@@ -0,0 +1,133 @@
+"""
+NCL_polyg_18.py
+==============
+This script illustrates the following concepts:
+   - Adding lines, markers, and polygons to a map
+   - Drawing lines, markers, polygons, and text in inset axes
+
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/polyg_18.ncl
+    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/polyg_18_2_lg.png
+"""
+
+###############################################################################
+# Import packages:
+# ----------------
+import cartopy.crs as ccrs
+import matplotlib.pyplot as plt
+import cartopy
+from geocat.viz import util as gvutil
+import matplotlib.patches as mpatches
+from cartopy.mpl.ticker import LatitudeFormatter, LongitudeFormatter
+import numpy as np
+###############################################################################
+# Define helper function to remove ticks/frames from axes
+
+
+def removeTicks(axis):
+    axis.get_xaxis().set_visible(False)
+    axis.get_yaxis().set_visible(False)
+
+
+###############################################################################
+# Plot map, markers, and polygons
+
+# Set size of figure
+fig = plt.figure(figsize=(10, 10))
+
+# Make grid on figure with 2 rows, 1 column
+grid = plt.GridSpec(2, 20, hspace=0.2, wspace=0.2, figure=fig)
+
+# Make subplot for map
+ax = plt.subplot(grid[:-1, 1:], projection=ccrs.PlateCarree())
+
+# Add continents
+continents = cartopy.feature.NaturalEarthFeature(
+        name="coastline",
+        category="physical",
+        scale="50m",
+        edgecolor="None",
+        facecolor="lightgray",
+)
+ax.add_feature(continents)
+
+# Set map extent
+ax.set_extent([-180, 180, -90, 90], crs=ccrs.PlateCarree())
```

Comment:
If you want to see the whole world, you can use `ax.set_global()` instead. It will show everything that a projection encompasses. 
```suggestion
ax.set_global()
```